keel-api/src/test/kotlin/com/netflix/spinnaker/keel/ec2/normalizers/ApplicationLoadBalancerAvailabilityZonesResolverTests.kt

```diff
@@ -0,0 +1,47 @@
+package com.netflix.spinnaker.keel.ec2.normalizers
```

Comment:
I thought you changed the package name to `resolvers`?

Reply:
But I haven't merged that PR yet!

Reply:
Oh, yeah. Doh! 🤦‍♂